sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

```diff
@@ -0,0 +1,660 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics;
+
+import com.azure.ai.textanalytics.models.DetectLanguageInput;
+import com.azure.ai.textanalytics.models.DetectLanguageResult;
+import com.azure.ai.textanalytics.models.DocumentResultCollection;
+import com.azure.ai.textanalytics.models.KeyPhraseResult;
+import com.azure.ai.textanalytics.models.LinkedEntityResult;
+import com.azure.ai.textanalytics.models.NamedEntityResult;
+import com.azure.ai.textanalytics.models.TextAnalyticsRequestOptions;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.ai.textanalytics.models.TextSentimentResult;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+
+import java.util.List;
+
+/**
+ * This class provides a synchronous client that contains all the operations that apply to Azure Text Analytics.
+ * Operations allow by the client to detect language, recognize entities, recognize pii entities,
+ * recognize linked entities, and analyze sentiment for a text input or a list of text input.
+ *
+ * <p><strong>Instantiating a synchronous Text Analytics Client</strong></p>
+ * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.instantiation}
+ *
+ * <p>View {@link TextAnalyticsClientBuilder this} for additional ways to construct the client.</p>
+ *
+ * @see TextAnalyticsClientBuilder
+ */
+@ServiceClient(builder = TextAnalyticsClientBuilder.class)
+public final class TextAnalyticsClient {
+    private final TextAnalyticsAsyncClient client;
+    private final String defaultCountryHint;
+    private final String defaultLanguage;
```

Comment:
Why do we need these? We can get it from the async client.

Reply:
These value will get from the the async client, but since they are used multiple times in client. Creating a local value for each will be good?

Reply:
I see it being used in a couple of places but those can be removed by just using the async equivalent counterparts.